### ISSUE TITLE
The minimal example for CTC

### ISSUE THREAD

**Issue by jjery2243542** [speechbrain/speechbrain#108]
1. Now the param.yaml is using spk_id as label. This is weird.
2. In my opinion, the minimal example could use training data to validation and test so that we could know whether it is overfitting or not.
